ext/native/thin3d/VulkanQueueRunner.cpp

```diff
@@ -488,6 +495,84 @@ void VulkanQueueRunner::ApplyMGSHack(std::vector<VKRStep *> &steps) {
 	}
 }
 
+void VulkanQueueRunner::ApplySonicHack(std::vector<VKRStep *> &steps) {
+	// We want to turn a sequence of render(3),render(1),render(6),render(1),render(6),render(1),render(3) to
+	// render(1), render(1), render(1), render(6), render(6), render(6)
```

Comment:
This one is definitely more gross, heh.  Reminds me of a certain other emulator's "skip codes"...

-[Unknown]